astropy/modeling/fitting.py

```diff
@@ -30,6 +30,7 @@
 import warnings
 
 from functools import reduce
+from pkg_resources import iter_entry_points
```

Comment:
Yes, it comes with setuptools. We don't explicitly list setuptools as a required dependency at the moment though. However, I think we could easily make a case that it should be. It would make sense to send an email to astropy-dev about this.